### PR TITLE
Add http_rgb platform

### PR DESCRIPTION
**Pull request by robhowlett**
**Description:**

Add support for hobby lights using simple HTTP API

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#9849

### CODE REVIEW

File: source/_components/light.http_rgb.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+To enable `http_rgb` in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: http_rgb
+    host: IP_ADDRESS
+```
+
+Configuration variables:
```
Comment (fabaff) [inline source/_components/light.http_rgb.markdown]: Please use the [new style](https://home-assistant.io/developers/documentation/create_page/#configuration).

File: source/_components/light.http_rgb.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_iot_class: "Local Polling"
+ha_release: 0.55
+---
+
+The `http_rgb` light platform lets you control lights that use a simple [HTTP API protocol](git@github.com:robhowlett/http-rgb).
```
Comment (fabaff) [inline source/_components/light.http_rgb.markdown]: This should point to the protocol but it goes to the binding repository. And we don't want to clone the repo here.

File: source/_components/light.http_rgb.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+title: "http_rgb"
+description: "Instructions how to setup http_rgb lights within Home Assistant."
+date: 2017-10-13 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_iot_class: "Local Polling"
+ha_release: 0.55
```
Comment (fabaff) [inline source/_components/light.http_rgb.markdown]: 0.55 was released last Saturday.

File: source/_components/light.http_rgb.markdown [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: "http_rgb"
```
Comment (fabaff) [inline source/_components/light.http_rgb.markdown]: Please stick to a name. So far there is `http_rgb`, `HTTP_RGB` and `HTTP_RGB!`. `http_rgb` is the platform name.

Comment (robhowlett) [reply source/_components/light.http_rgb.markdown]: Agreed - I've updated my pull request on home-assistant/home-assistant to reflect a consistent naming standard.

File: source/_components/light.http_rgb.markdown [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_iot_class: "Local Polling"
+ha_release: 0.56
+---
+
+The `http_rgb` light platform lets you control lights that use a simple [HTTP API protocol](https://github.com/robhowlett/http_rgb/README.md).
```
Comment (fabaff) [inline source/_components/light.http_rgb.markdown]:
```bash
$ wget https://github.com/robhowlett/http_rgb/README.md
--2017-10-14 15:14:47--  https://github.com/robhowlett/http_rgb/README.md
Resolving github.com (github.com)... 192.30.253.112, 192.30.253.113
Connecting to github.com (github.com)|192.30.253.112|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-10-14 15:14:48 ERROR 404: Not Found.
```

**Comment by robhowlett** [comment]
Closing, along with home-assistant/home-assistant#9849, following comments to convert to REST light platform. I will work on this and resubmit in the future.

**Pull request closed without merging**